src/test/spec/unified_runner/operation.rs

```diff
@@ -101,6 +101,12 @@ pub(crate) trait TestOperation: Debug + Send + Sync {
     fn returns_root_documents(&self) -> bool {
         false
     }
+
+    /// If this operation specifies entities to create, returns those entities. Otherwise,
+    /// returns None.
+    fn as_test_file_entities(&self) -> Option<&Vec<TestFileEntity>> {
```

Comment:
oh yeah, I think that name was a carry over from a previous version of the method which returned the `CreateEntities` operation itself rather than just its entities. renamed.